### PR TITLE
Add dev-cold environment for dev environment without automatic refresh

### PR DESCRIPTION
**Pull request by vinteo**
Add `dev-cold` environment for dev environment without automatic refresh

### CODE REVIEW

File: cfg/base.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ module.exports = {
   devServer: {
     contentBase: './src/',
     historyApiFallback: true,
+    host: host,
```
Comment (camjc) [inline cfg/base.js]: we can probably shorthand all these `{foo: foo}`s

File: README.md [hunk 2]
```diff
@@ -40,6 +40,10 @@ Commands
 
 npm start # or npm run serve
 
+# Start the dev-server without automatic refresh
```
Comment (camjc) [inline README.md]: refresh isn't strictly accurate, we should probably say hot loading to be accurate.

**Comment by camjc** [comment]
Looking good :)

**Comment by vinteo** [comment]
@camjc merge?

File: README.md [hunk 3]
```diff
@@ -40,9 +40,9 @@ Commands
 
 npm start # or npm run serve
 
-# Start the dev-server with the dist version
+# Start the dev-server without automatic refresh
 
-npm run serve:dist
+npm run serve:cold
```
Comment (camjc) [inline README.md]: `npm run start:cold`

File: package.json [hunk 4]
```diff
@@ -11,12 +11,12 @@
   "main": "dist/alexandria-main.js",
   "scripts": {
     "start": "node server.js --env=dev",
+    "start:cold": "node server.js --env=dev-cold",
     "test": "karma start",
     "test:watch": "karma start --autoWatch=true --singleRun=false",
     "codecov": "cat coverage/lcov.info | codecov",
     "posttest": "npm run lint && npm run sass-lint && npm run jscs-lint && npm run check-commit",
     "serve": "node server.js --env=dev",
```
Comment (camjc) [inline package.json]: may as well kill this too while we're here.

**Comment by camjc** [comment]
Merging!

**Pull request merged**